src/languages/es.ts

```diff
@@ -654,14 +654,26 @@ export default {
         },
         waitingOnEnabledWallet: ({submitterDisplayName}: WaitingOnBankAccountParams) => `Inició el pago, pero no se procesará hasta que ${submitterDisplayName} active su Billetera`,
         enableWallet: 'Habilitar Billetera',
-        hold: 'Hold',
-        holdEducationalTitle: 'Esta solicitud está en',
-        whatIsHoldTitle: '¿Qué es Hold?',
-        whatIsHoldExplain: 'Hold es nuestra forma de agilizar la colaboración financiera. ¡"Rechazar" es tan duro!',
-        holdIsTemporaryTitle: 'Hold suele ser temporal',
-        holdIsTemporaryExplain: 'Debido a que hold se utiliza para aclarar confusión o aclarar un detalle importante antes del pago, no es permanente.',
+        holdRequest: 'Bloquear solicitud de dinero',
+        unholdRequest: 'Desbloquear solicitud de dinero',
+        explainHold: 'Explica la razón para bloquear esta solicitud.',
+        reason: 'Razón',
+        holdReasonRequired: 'Se requiere una razón para bloquear.',
+        requestOnHold: 'Este solicitud está bloqueada. Revisa los comentarios para saber como proceder.',
+        confirmApprove: 'Confirma que quieres aprobar',
+        confirmApprovalAmount: 'Aprobar el total o solo la parte no bloqueada.',
+        confirmPay: 'Confirma que quieres pagar',
+        confirmPayAmount: 'Pagar todos los gastos por cuenta propia o solo el monto no bloqueado.',
+        payOnly: 'Solo pagar',
+        approveOnly: 'Solo aprobar',
+        hold: 'Bloqueada',
+        holdEducationalTitle: 'Esta solicitud está',
+        whatIsHoldTitle: '¿Qué es Bloquear?',
+        whatIsHoldExplain: 'Bloquear es nuestra forma de agilizar la colaboración financiera. ¡"Rechazar" es tan duro!',
+        holdIsTemporaryTitle: 'Bloquear suele ser temporal',
+        holdIsTemporaryExplain: 'Se utiliza bloquear para aclarar confusión o aclarar un detalle importante antes del pago, no es permanente.',
         deleteHoldTitle: 'Eliminar lo que no se pagará',
-        deleteHoldExplain: 'En el raro caso de que algo se ponga en hold y no se pague, la persona que solicita el pago debe eliminarlo.',
+        deleteHoldExplain: 'En el raro caso de que algo se bloquear y no se pague, la persona que solicita el pago debe eliminarlo.',
```

Comment:
Just not to skip this, have you confirmed translations on #expensify-open-source?

Reply:
This actually comes from Expensify internal engineers 😄